html/exercise-04/script.js

```diff
@@ -0,0 +1,22 @@
+const user = {
+    name: "Piyush Sharma",
+    designation: "Senior Software Engineer",
+    company: "Infuse Consulting",
+    hobbies: ["Reading", "Listening to music", "Collecting stamps"]
+}
+
+/**
+ * Use destructuting to log the following
+*/
+
+const printUserProfile = () => {
+    // Piyush Sharma is a Senior Software Engineer at Infuse Consulting. He likes Reading, Listening to music and Collecting stamps
+    const { name:Name, designation:jobTitle, company:company, hobbies:hobbies } = user;
```

Comment:
why assign it to new variable of `Name` here, we are moving away from the camelCase convention used in JavaScript